src/server.ts

```diff
@@ -101,7 +100,6 @@ export default function runServer({ port = 8080 } = { port: 8080 }) {
 				// Serve files!
 				//
 				if (request.url.startsWith('/test')) {
-					//	console.log('Serving .' + request.url);
```

Comment:
Shouldn't this console log have been uncommented instead of removed, all other files which are being served are being logged.

Reply:
I added it as uncommented